gamestonk_terminal/cryptocurrency/defi/terramoney_fcd_view.py

```diff
@@ -0,0 +1,301 @@
+"""Terra Money FCD view"""
+__docformat__ = "numpy"
+
+import os
+from tabulate import tabulate
+import matplotlib.pyplot as plt
+from matplotlib import ticker, dates as mdates
+from gamestonk_terminal.cryptocurrency.defi import terramoney_fcd_model
+from gamestonk_terminal.helper_funcs import (
+    export_data,
+    long_number_format,
+    plot_autoscale,
+    rich_table_from_df,
+)
+from gamestonk_terminal.cryptocurrency.dataframe_helpers import (
+    prettify_column_names,
+    very_long_number_formatter,
+)
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.config_plot import PLOT_DPI
+from gamestonk_terminal.rich_config import console
+
+
+def display_account_staking_info(
+    address: str = "", top: int = 10, export: str = ""
+) -> None:
+    """Display staking info for provided terra account address [Source: https://fcd.terra.dev/swagger]
+
+    Parameters
+    ----------
+    address: str
+        terra blockchain address e.g. terra1jvwelvs7rdk6j3mqdztq5tya99w8lxk6l9hcqg
+    top: int
+        Number of records to display
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+
+    df, report = terramoney_fcd_model.get_staking_account_info(address)
+    console.print(report)
```

Comment:
```
Overview: 
   Address: terra1glpjvujn0gjdjqra2ejgu7um2yd9fu63n7h29u
   Available Luna: 0.15
   Delegated Luna: 0.0
   Total Rewards:  0.0

Delegations: 
```

This is a small detail and i'm not sure about this but what about checking if df is empty to display "No delegations found"?